src/main/java/io/github/sornerol/chess/pubapi/domain/game/PlayerDetail.java

```diff
@@ -36,4 +37,12 @@ public class PlayerDetail {
      */
     @JsonProperty("username")
     private String username;
+    
+    /**
+     * The uuid of the Player
+     */
+    //TODO Check if the String can replace throw a UUID
```

Comment:
I don't understand this TODO comment. Can you elaborate on what you mean by "Check if the String can replace throw a UUID"?

Reply:
I mean if we can i replace with the class „UUID“ form the java.lang packet

Reply:
Can you test this using UUID instead of String? I believe Jackson supports serialization/deserialization of UUIDs by default.